### DISCUSSION TITLE
How to change the label for the Create Button in Resource?

### DISCUSSION THREAD

**Discussion by JenuelDev** [laravel/nova-issues#3332] [Q&A] [answered]
I was hoping to change this `create Image` to `Add New Image`
![image](https://user-images.githubusercontent.com/31676163/114005580-3f534100-9892-11eb-8cdb-17fb924ccc23.png)

**Comment by crynobone** [comment, marked as answer]
Override the method from the desired resource class.

```php
    /**
     * Get the text for the create resource button.
     *
     * @return string|null
     */
    public static function createButtonLabel()
    {
        return __('Create :resource', ['resource' => static::singularLabel()]);
    }
```

**Reply by JenuelDev** [reply]
By the way, thanks.. Ill try it out later :)

**Reply by Muetze42** [reply]
This only changes the button. In the form it is still "`Create...`".

Is it possible to correct that there as well?

**Reply by Gismo1337** [reply]
@Muetze42 

```
public static function createButtonLabel()
    {
        return 'YourButtonTextHere';
    }

```

**Reply by eslamabdallah74** [reply]
is there a way to change the URL of the button ,
for example I created a custom page for create so how can I make the create button go to the new URL?

**Comment by culturalaspect** [comment]


**Reply by ziming** [reply]
this is not filament

**Comment by Paulsky** [comment]
> This only changes the button. In the form it is still "`Create...`".
> 
> Is it possible to correct that there as well?

@Muetze42 I don't think so. Which is hilarious in my opinion. Want to change the button text? Okay. Want to change the create title? Not possible 😂. I had to hack it with CSS:

```
h1[dusk="create-RESOURCENAME-panel-heading"] {
  visibility: hidden;
  font-size: 0;
  position: relative;
}


h1[dusk="create-RESOURCENAME-panel-heading"]::after {
  content: 'Add New Image';
  visibility: visible;
  font-size: 1.25rem;
  position: absolute;
  top: 0;
  left: 0;
}
```
